org.eventb.emf.core.edit/build.properties

```diff
@@ -1,12 +1,16 @@
-# Copyright (c) 2006-2014
-# University of Southampton, Heinrich-Heine University Dusseldorf and others.
-# All rights reserved. This program and the accompanying materials  are made
-# available under the terms of the Eclipse Public License v1.0 which accompanies this 
-# distribution, and is available at http://www.eclipse.org/legal/epl-v10.html
-# 
+###############################################################################
+# Copyright (c) 2014, 2015 University of Southampton.
```

Comment:
> And it should be 2022?

not unless there has been a change to that file (other than copyright) .. the copyright tool uses the repo to determine the date.. but i left it set to update all copyrights because some were still missing the 2.0 EPL.. agree we should not drop HHU and others though